sprocs/variants_select_submission_for_manual_grading.sql

```diff
@@ -0,0 +1,34 @@
+DROP FUNCTION IF EXISTS variants_select_submission_for_manual_grading(bigint,boolean);
```

Comment:
@mwest1066 

Just a note that this is duplicate code taken from `variants_select_submission_for_grading.sql`, except I took these two lines out of the former block from the latter block:

```
    IF submission.score IS NOT NULL THEN RETURN; END IF; -- already graded
    IF submission.grading_requested_at IS NOT NULL THEN RETURN; END IF; -- grading is in progress
```

```
    -- does the most recent submission actually need grading?
    IF submission.score IS NOT NULL THEN RETURN; END IF; -- already graded
    IF submission.grading_requested_at IS NOT NULL THEN RETURN; END IF; -- grading is in progress
    IF submission.broken THEN RETURN; END IF;
    IF NOT submission.gradable THEN RETURN; END IF;
```

I just wanted to take these out so that an already autograded item will be queried. I can refine this my adding a branch for manual grading logic to the original `variants_select_submission_for_grading.sql` file.

Just wanted to give you a heads-up about the duplicate code.

Reply:
Removed use of this file. Brought `instance_question_select_manual_grading_objects.sql` into existence with necessary last submission fetch.